src/main/java/eu/dissco/core/translator/terms/specimen/identification/AbcdTypeInformation.java

```diff
@@ -0,0 +1,68 @@
+package eu.dissco.core.translator.terms.specimen.identification;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import eu.dissco.core.translator.schema.DigitalSpecimen;
+import eu.dissco.core.translator.schema.Identifications;
+import eu.dissco.core.translator.terms.Term;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AbcdTypeInformation extends Term {
+
+  private static final Logger log = LoggerFactory.getLogger(AbcdTypeInformation.class);
+
+  public void addTypeInformation(DigitalSpecimen ds, JsonNode unit, ObjectMapper mapper) {
+    var typeDesignationNodes = getTypeDesignationNodes(unit, mapper);
+    if (typeDesignationNodes.isEmpty()) {
+      return;
+    }
+    var identifications = ds.getDwcIdentification();
+    if (typeDesignationNodes.size() == 1 && identifications.size() == 1) {
```

Comment:
I don't think you need to check this condition. the code in the else { } block would handle it just as well.

Reply:
No the else block checks based on typifiedName, this one only checks if there is 1 name and 1 typeStatus it must be ok even if there is no typified name or if they don't match.